yt_dlp/networking/common.py

```diff
@@ -421,6 +421,9 @@ def _prepare_headers(self, request):
         if self.SUPPORTED_ENCODINGS and 'Accept-Encoding' not in request.headers:
             request.headers['Accept-Encoding'] = ', '.join(self.SUPPORTED_ENCODINGS)
 
+        if 'Accept-Encoding' not in request.headers:
+            request.headers['Accept-Encoding'] = 'identity'
+
```

Comment:
Isn't this the default (ie if no 'Accept-Encoding' header is sent)?

Reply:
there is no guarantee as the backend may not be http lib based

Reply:
Oh, sure. The *protocol* default but not necessarily the random back-end default. Worth a comment?